po/POTFILES.in

```diff
@@ -9,6 +9,7 @@ js/extensionPrefs/main.js
 js/gdm/authPrompt.js
 js/gdm/loginDialog.js
 js/gdm/util.js
+js/misc/paygManager.js
```

Comment:
Why is this being added to `POTFILES.in` in this commit? The commit doesn’t add any translatable strings, as far as I can see.

Reply:
Another leftover from a previous version where I did actually had translatable strings here, in the -now gone- `proxyToPaygError()` function.